cli/config-cli/src/main/java/com/quorum/tessera/config/cli/TesseraCommand.java

```diff
@@ -71,6 +71,16 @@ private TesseraCommand(final KeyPasswordResolver keyPasswordResolver, Validator
       description = "Start Tessera in recovery mode")
   private boolean recover;
 
+  @CommandLine.Option(
+      names = {"--outputServerURIs"},
```

Comment:
Would it not be enough to just set the output path? Setting the path could mean, yes I do want the path, and here is the path?